lib/common/transforms.js

```diff
@@ -832,6 +832,7 @@ export default {
       }
       const parsedVal = parseFloat(nonParsed);
       if (isNaN(parsedVal)) throwSizeError(token.name, nonParsed, 'rem');
+      if (parsedVal === 0) return '0';
```

Comment:
```suggestion
      if (parsedVal === 0) return Number.isInteger(nonParsed) ? 0 : '0';
```

Let's check that if the token value is the number 0, we leave it as a number. Please add a test for this as well :)